src/app/docs/file-attachment/file-attachment-docs.component.html

```diff
@@ -0,0 +1,189 @@
+          captionType="success"
+          imageSource="~/assets/img/guidelines/file-attach/use-multi.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline>
+        <p>
+          Use the file attachment element when users need to add metadata about uploaded files.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use a file attachment element to add metadata about uploaded files."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/file-attach/use-metadata.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p class="sky-docs-text-block">
+          Don't use the file attachment element when users only need to upload a single local file. Use the <a href="https://developer.blackbaud.com/skyux-forms/docs/single-file-attachment">single file attachment component</a> instead.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        imagePath="~/assets/img/guidelines/file-attach/anatomy.png"
+      >
+        <sky-docs-anatomy-item>Option to upload local files</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>File name</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>File size</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>File preview</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Delete button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Option to link to external files</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">User-entered metadata</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Link to external files"
+      >
+        <p class="sky-docs-text-block">
+          You can indicate whether to include the option to attach external files using their URLs.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Metadata"
+      >
+        <p class="sky-dpcs-text-block">
+          You can include inputs for users to add metadata about file attachments.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Validation"
+      >
+        <p class="sky-docs-text-block">
+          You can specify parameters for valid files, including file types, minimum and maximum sizes, and custom validation.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Behavior & States"
```

Comment:
`&` should be `and` (Adam and Todd implemented this with the ampersand, but we recently updated our guidance to not use ampersands), and `states` should be lowercase.

Reply:
Done